app/src/main/java/com/bashkevich/androidfundamentals/MainActivity.kt

```diff
@@ -17,6 +21,18 @@ class MainActivity : AppCompatActivity() {
                 add<FragmentMoviesList>(R.id.main_container)
             }
         }
+
+        lifecycleScope.launchWhenCreated {
+            val constraints =
```

Comment:
How about to extract it to some Manager and call it here?

Reply:
I think it's not necessary because there is  not so many code

Reply:
The point of extraction is not to minimise the code but rather to separate responsibilities.